ReviewMonitor/Reviews List View Controller/Editor View Controller/ResponseEditorViewController.swift

```diff
@@ -24,6 +24,13 @@ class ResponseEditorViewController: UIViewController {
         navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Post", style: .plain, target: self, action: #selector(postTapped))
     }
 
+    func textViewDidBeginEditing(_ textView: UITextView) {
```

Comment:
What is this change for?

Reply:
I removed it just now. It was when I was looking for placeholder. See slack. What we would do later

Reply:
In that case, you should probably use separate branches for different features instead of working directly on `master`.

Reply:
Yes...

Reply:
Updated